src/components/molecules/CheckboxGroupField/CheckboxGroupField.md

```diff
@@ -0,0 +1,78 @@
+### Summary
+
+`<CheckboxGroupField />` is a convenient wrapper that renders group of `<CheckboxField>`s:
+
+⚠️ &nbsp;Note that the `name` for each item has to be unique as it determines the `id`
+and `htmlFor` prop fof each checkbox field.
```

Comment:
`for each checkbox`

You might want to install [vscode spell check](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker), my life got easier after it 😆 💅🏻

Reply:
installed and fixed :)